docs/assets/css/site.css

```diff
@@ -0,0 +1,520 @@
+.top-header .menu-toggle {
+    top: 10px;
+}
+
+@media (max-width: 992px) {
+    .top-header .container .menu {
+        top: 52px;
+        background-color: #171717;
+    }
+}
+
+/*
+    Splash Page
+*/
+
+.splash-page {
+    background-color: #e5e5e5;
+}
+
+.splash-page .splash-header {
+    padding-top: 54px;
+    background: #000;
+    box-shadow: none;
+    height: 500px;
+    border-bottom: 1px solid #0a0f15;
+}
+
+.splash-page .splash-header .caf-brand {
+    background: url(../img/autoscaler-header.png), url(../img/autoscaler-header-image-2.png), #000;
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: left bottom;
+    padding-right: 15px;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+    height: 100%;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.splash-page .splash-header .caf-brand .title {
+    font-size: 50px;
+    text-transform: uppercase;
+    margin-bottom: 20px;
+}
+
+.splash-page .splash-header .caf-brand .slogan {
+    font-family: Metric;
+}
+
+@media (min-width: 768px) {
+    .splash-page .splash-header .caf-brand {
+        width: 750px;
+    }
+}
+
+@media (min-width: 992px) {
+    .splash-page .splash-header .caf-brand {
+        width: 970px;     
+    }
+}
+
+@media (min-width: 1200px) {
+    .splash-page .splash-header .caf-brand {
+        width: 1400px;
+        justify-content: flex-end;        
+    }
+}
+
+
+.splash-page .splash-header .caf-brand .download-caf {
+    background-color: #b8ffd8;
+    color: #fff;
+    padding: 15px 65px;
+    height: auto;
+    margin-top: 0;
+    transition: background-color 0.3s ease-in-out;
+}
+
+.splash-page .splash-header .caf-brand .download-caf:focus, .splash-page .splash-header .caf-brand .download-caf:visited, .splash-page .splash-header .caf-brand .download-caf:active {
+	color: #000;
+}
+
+.splash-page .splash-header .caf-brand .download-caf:hover {
+    background-color: #6EAF8C;
+}
+
+.spash-page {
+	background-color: #e5e5e5;
+}
+.splash-page hr {
+    border-top: 1px solid #323d43;
+    max-width: 1170px;
+    margin: 0 auto;
+    width: 100%;
+}
+
+.splash-page .social-card .icon {
+    color: #74E291;
+}
+
+.splash-page .social-card .title {
+    color: #000;
+}
+
+.splash-page .social-card .subtitle {
+    color: #000;
+}
+
+.splash-page .social-card .link {
+    color: #b8ffd8;
+}
+
+.splash-page .social-card .link:hover {
+    color: #fff;
+}
+
+.splash-page .section-title {
+    color: #000;
+}
+
+.splash-page .section-subtitle {
+    font-size: 20px;
+    color: #000;
+}
+
+.feature-list .feature .title {
+    color: #000;
+}
+
+.feature-list .feature .subtitle {
+    color: #000;
+}
+
+/*
+    Default Layout
+*/
+
+.content-container {
+    padding-top: 54px;
+}
+
+.content-container .docs-content h1, 
+.content-container .docs-content h2,
+.content-container .docs-content h3 {
+    font-family: Metric;
+    color: #fff;
+    border-color: #171717;
```

Comment:
The horizontal lines under some headings are hard to make out agianst the dark background. Can you change the border-colour to the same colour as the text so they are easier to see?